GroceryStore/Product.m

```diff
@@ -0,0 +1,42 @@
+//
+//  Product.m
+//  GroceryStore
+//
+//  Created by Jilian Scott on 5/13/15.
+//  Copyright (c) 2015 Vokal. All rights reserved.
+//
+
+#import "Product.h"
+
+@interface Product ()
+
+@property (strong, nonatomic) NSString *productName;
+
+@property (strong, nonatomic) NSString *quantity;
+
+@end
+@implementation Product
+
+- (id)initWithProductName:(NSString *)productName andQuantity:(NSNumber *)quantity
+{
+    self = [super init];
+    if (self) {
+        self.productName = productName;
```

Comment:
when setting properties in an init method, you should set the ivar directly: `_productName = productName;`
See: "Don’t Use Accessor Methods in Initializer Methods and dealloc" here:
https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/MemoryMgmt/Articles/mmPractical.html
and this: http://qualitycoding.org/objective-c-init/